### ISSUE TITLE
Add support for starting a debug session from the command line

### ISSUE THREAD

**Issue by fabiospampinato** [Microsoft/vscode#46496]
I like the debugger that comes with Code, and I'd like to use it more often, but in some cases starting it up is cumbersome.

#### My use case:

- I install an NPM package globally (`npm i -g foo`).
- I use it a bit and for some reason I want to start up a debug session to find what exactly happens when I execute `foo --flag1` in the terminal.

As far as I know the only way to do that would be to create a launch configuration file, which would look like this:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "/whatever/foo.js",
            "args": ["--flag1"]
        }
    ]
}
```

I have some problems with this approach:

- I don't want to pollute every single project I want to debug with launch configuration files.
- I can already do `node --inspect-brk /whatever/foo.js --flag1` and with the help of [NiM](https://chrome.google.com/webstore/detail/nodejs-v8-inspector-manag/gnhhdgbaldcilmgcpfddgdbkhjohddkj) I immediately get the devtools open, why isn't it as easy, or easier, to do the same with Code?
- Never mind if you also want to debug `foo --flag2`, go write another configuration for it.

Basically always requiring the creation of a launch configuration files leads to an awful user experience for me.

#### Proposed solution:

Node has the `--inspect` and `--inspect-brk` command line flags, I think similar flags should be added to Code itself. Code already comes with the `--inspect-extensions` and `--inspect-brk-extensions` flags, so the lack of those other flags seems like a major overlooking to me.

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/10979
Thanks bot!
